### ISSUE TITLE
Remove Events and Glossary drawers from the site header

### ISSUE THREAD

**Issue by mcharg** [ONRR/onrr.gov-site#773]
And link to new static pages to fix accessibility issues with the drawers.

**Comment by mcharg** [comment]
This is updated in dev.

@Maroyafaied @jlouis-onrr Take a look and let me know if it looks good to push live.

**Comment by Maroyafaied** [comment]
@mcharg I still see them in dev, am I looking at the right one in the header? Should these icons still be there?
![image](https://user-images.githubusercontent.com/40175688/121546957-f66e5480-c9c8-11eb-824e-c3cfc3ed003f.png)

**Comment by mcharg** [comment]
@Maroyafaied The icons now link to the static pages instead of open the drawer.

**Comment by Maroyafaied** [comment]
@mcharg makes sense now.. looks good. Thanks

**Comment by mcharg** [comment]
Updated everywhere on the live site except the two pages updated in #776.  Those will get updated when I get approval to push those changes live.

**Comment by Toddsand21** [comment]
Is there something else I should be proofing before it is pushed?  I know we had 2 or 3 items on the Tribal Consultation page and the Tribal Assistance page had 1 item (link to Tribal Liaison) in upper right and the contacts on the bottom of page updated., Then we'll get to the deletions of the 2019 Scheduled Consultation which I don't think I've submitted yet have I?  I can't view the changes of these, any suggestions?

**Comment by mcharg** [comment]
@Toddsand21 The Tribal Assistance changes are in issue #776.  I haven't seen any issues related to the Tribal Consultation page, though I did change the email address on that page to Bethany's as mentioned in #776.

Tribal Assistance in dev:
http://2017onrr/IndianServices/tribal-assistance.htm

Tribal Consultation in dev:
http://2017onrr/IndianServices/consultation.htm

**Comment by mcharg** [comment]
@Toddsand21 This issue just changed the events drawer to a page without any content changes.  The drawer was causing major issues for screen readers.

New Events page:
http://2017onrr/events.htm
